opsdroid/events.py

```diff
@@ -134,6 +137,8 @@ def __init__(
         # Inherit the user, target and event from the event we are responding
         # to if they are not explicitly provided by this Event
         event.user = event.user or self.user
+        event.user_id = event.user_id or self.user_id or event.user
+        event.user = event.user or self.user
```

Comment:
You have this line twice now.
```suggestion
```

Reply:
Sorry, removed it. Thanks for suggestion